oak_containers_system_image/Dockerfile

```diff
@@ -3,6 +3,14 @@ FROM clearlinux:base
 # Remove .dockerenv as we will not run this image in an actual Docker container.
 RUN rm /.dockerenv
 
+# Install Clear Linux's containers-basic bundle. This gives an installation of
+# docker to run containers. This is a starting point only, we should switch to
+# a more lightweight runtime later.
+RUN swupd bundle-add containers-basic
+
+# Configure systemd to run docker at startup
+RUN systemctl enable docker
```

Comment:
We can already use `systemctl` during the build system, it creates a service config file that is then baked into the file system.